code/game/jobs/job/security.dm

```diff
@@ -182,6 +182,13 @@
 				H.equip_or_collect(new /obj/item/clothing/suit/storage/det_suit(H), slot_wear_suit)
 				H.equip_or_collect(new /obj/item/clothing/head/det_hat(H), slot_head)
 				H.equip_or_collect(new /obj/item/clothing/shoes/brown(H), slot_shoes)
+			if("Private Eye")
```

Comment:
don't paste this, just add 

`,"Private Eye"` after Gumshoe above

Reply:
Done